rust-runtime/aws-smithy-http-server/src/build_modifier.rs

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use crate::operation::Operation;
+
+pub trait BuildModifier<Protocol, Op, S, L> {
```

Comment:
Missing Rust docs?

Reply:
Removed this completely, will add it back in a corollary PR.